cclib/parser/orcaparser.py

```diff
@@ -2810,12 +2815,11 @@ def _append_scfvalues_scftargets(self, inputfile, line):
                 rmsDP_value = float(line.split()[4])
                 rmsDP_target = float(line.split()[7])
             else:
-                rmsDP_value = self.scfvalues[-1][-1][2]
-                rmsDP_target = self.scftargets[-1][2]
```

Comment:
This line is the cause of the error. When the RMS density target is not available at the end of the current SCF cycle, it uses the value from the previous cycle. This is not desirable for at least two reasons:
 1) There is no guarantee the RMS density target was available in the last cycle (it could be missing in the very first)
 2) The logic here doesn't make sense to me. It's certainly rare for the SCF targets to change mid-job, but there's no reason this couldn't happen. Orca has a complicated SCF solver (I think actually multiple solvers?) that likely doesn't use all of the targets all of the time. If a target isn't reported, then it seems more likely to me that it wasn't used, rather than Orca forgot to mention it (but perhaps I'm being to generous to Orca...)

Reply:
Yes, any program that can switch between solvers (Orca, Q-Chem, Psi4, even Gaussian) will have this problem.